src/utils/walletConnect.js

```diff
@@ -141,18 +144,19 @@ export const mapCallRequestToTransactionPayload = (
     to = methodToAddress;
   }
 
-  const ethAssetData = getAssetData(accountAssets, supportedAssets, ETH);
+  const { chainId } = callRequest;
+  const chain = chainFromChainId[chainId];
+  const chainNativeSymbol = nativeAssetSymbolPerChain[chain];
+  const chainNativeDecimals = nativeAssetDecimalsPerChain[chain];
+
+  const nativeAssetData = getAssetData(accountAssets, supportedAssets, chainNativeSymbol);
```

Comment:
Will it work for BNB token, atm supported assets do not contain BNB.

Reply:
It won't. It will work once we add support to it.